src/Storage/Controllers/MessageboxInstancesController.cs

```diff
@@ -569,5 +573,16 @@ private async Task<ActionResult> ProcessQueryResponse(InstanceQueryResponse quer
 
             return Ok(authorizedInstances);
         }
+
+        private void AddQueryModelToTelemetry(MessageBoxQueryModel queryModel)
+        {
+            if (Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT").Equals("Development", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
Any reason for why we're excluding dev? Does it fail? Could we instead handle by checking the object returned by `Get<RequestTelemetry>()`?

Reply:
Can change it to check the object. We don't have the connection to AI when running locally.  We need to have some sort of check to avoid an exception being thrown.